Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/MSBuildTools.cs

```diff
@@ -46,8 +96,45 @@ public static MSBuildToolsConfig Load()
                 EditorJsonUtility.FromJsonOverwrite(File.ReadAllText(MSBuildSettingsFilePath), toReturn);
             }
```

Comment:
Does this succeed even if fields are missing based on version changes?

Reply:
As I understand, yes.

---
In reply to: [364491094](https://github.com/microsoft/MSBuildForUnity/pull/96#discussion_r364491094) [](ancestors = 364491094)